assets/app/scripts/controllers/settings.js

```diff
@@ -34,12 +34,19 @@ angular.module('openshiftConsole')
         angular.forEach(limitRange.spec.limits, function(limit) {
           limit.min = limit.min || {};
           limit.max = limit.max || {};
+          limit.default = limit.default || {};
           angular.forEach(limit.max, function(value, type) {
             limit.min[type] = limit.min[type] || "";
+            limit.default[type] = limit.default[type] || "";
```

Comment:
Will "default" as a keyword mess us up? I noticed the minifier switching to a quoted subscript access. Should we do the same thing in the javascript and in the template?